src/site/stages/build/plugins/inject-axe-core.js

```diff
@@ -36,7 +36,7 @@ function injectAxeCore(buildOptions) {
       );
 
       const executeAxeCheckScript = dom(
-        `<script type="text/javascript" src="/js/execute-axe-check.js"></script>`,
+        `<script type="text/javascript" src="/generated/execute-axe-check.js"></script>`,
```

Comment:
This is the one JS asset that we need webpack to copy over.

Reply:
I think this is fine, but it's a weird bit of semantics since Webpack didn't technically _generate_ anything here. Just copied over what was already there without even peaking at what it is. :man_shrugging:

Reply:
I agree with the weird semantics, but now I'm even questioning if this is the right thing to do.  Time for some more thinking.